### DISCUSSION TITLE
Is there any way to overwrite 'OEmbedAction' file?

### DISCUSSION THREAD

**Discussion by victore13** [awcodes/filament-tiptap-editor#502] [Q&A] [answered]
Hello! How's everything going?

I'm working with TipTapEditor and I need to modify the action logic of `OEmbedAction` file so that some checkboxes are always checked by default, among other tasks.

So, just like I did with the LinkAction file, I was hoping to go into the configuration file, overwrite a variable called 'oembed_action' with the value of the path of my custom file and work the desired functions on my own file. The problem is that this variable is not contemplated in their configuration file.

Is there a way to do this?

**Comment by awcodes** [comment, marked as answer]
At the moment, no, I'm open to a PR to allow it though. There was a recent PR to make the Grid Action override-able.

**Reply by victore13** [reply]
Great, I'll release a PR in the next few days to try and mitigate this issue.

Thank you very much for your response and understanding.
